app/controllers/pias_controller.rb

```diff
@@ -66,6 +103,29 @@ def import
 
   private
 
+  # return the params if it's not a user
+  def check_user_id(user_id)
+    if user_id.to_i.positive?
+      User.find(user_id.to_i)
+    else
+      user_id
+    end
+  end
+
+  def update_pia_user_field(field, role)
+    user = check_user_id(pia_params[field])
+    return unless user.is_a?(User)
```

Comment:
Always a new line after un return.
```suggestion
    return unless user.is_a?(User)

```